libsolidity/codegen/ABIFunctions.h

```diff
@@ -44,15 +44,16 @@ using TypePointers = std::vector<TypePointer>;
 /// multiple times.
 ///
 /// Make sure to include the result of ``requestedFunctions()`` to a block that
-/// is visible from the code that was generated here.
+/// is visible from the code that was generated here, or use named labels.
 class ABIFunctions
 {
 public:
-	~ABIFunctions();
-
-	/// @returns assembly code block to ABI-encode values of @a _givenTypes residing on the stack
+	/// @returns name of an assembly function to ABI-encode values of @a _givenTypes
 	/// into memory, converting the types to @a _targetTypes on the fly.
-	/// Assumed variables to be present: <$value0> <$value1> ... <$value(n-1)> <$headStart>
+	/// Parameters are: <$value0> <$value1> ... <$value(n-1)> <$headStart>
+	/// The values represent stack slots. If a type occupies more or less than one
```

Comment:
I would explain this generates a function which receives those stack slots from top to bottom, i.e. the parameter list is reversed compared to this.